pkgs/by-name/ve/vesktop/package.nix

```
@@ -125,7 +125,7 @@ stdenv.mkDerivation (finalAttrs: {
     ''
     + lib.optionalString stdenv.hostPlatform.isDarwin ''
       mkdir -p $out/{Applications,bin}
-      mv dist/mac*/Vesktop.App $out/Applications
+      mv dist/mac*/Vesktop.app $out/Applications
```

Does this work because of the case-insensitive filesystem?

To me, a more correct way would be this as it wouldn't infer the filename based on the source directory:

```suggestion
      mv dist/mac*/Vesktop.app $out/Applications/Vesktop.app
```

---

> Does this work because of the case-insensitive filesystem?

I think so, since there is no `.App`

<details>

```
Running phase: installPhase
---------------
dist/:
total 8
-rw-r--r--  1 _nixbld1 nixbld  920 Nov  3 12:27 builder-debug.yml
-rw-r--r--  1 _nixbld1 nixbld 1646 Nov  3 12:27 builder-effective-config.yaml
drwxr-xr-x 10 _nixbld1 nixbld  320 Nov  3 12:27 js
drwxr-xr-x  3 _nixbld1 nixbld   96 Nov  3 12:27 mac-arm64

dist/js:
total 4164
-rw-r--r-- 1 _nixbld1 nixbld 1275999 Nov  3 12:27 main.js
-rw-r--r-- 1 _nixbld1 nixbld 2827042 Nov  3 12:27 main.js.map
-rw-r--r-- 1 _nixbld1 nixbld    2290 Nov  3 12:27 preload.js
-rw-r--r-- 1 _nixbld1 nixbld    8068 Nov  3 12:27 preload.js.map
-rw-r--r-- 1 _nixbld1 nixbld    2542 Nov  3 12:27 renderer.css
-rw-r--r-- 1 _nixbld1 nixbld    5447 Nov  3 12:27 renderer.css.map
-rw-r--r-- 1 _nixbld1 nixbld   26140 Nov  3 12:27 renderer.js
-rw-r--r-- 1 _nixbld1 nixbld  100634 Nov  3 12:27 renderer.js.map

dist/mac-arm64:
total 0
drwxr-xr-x 3 _nixbld1 nixbld 96 Nov  3 12:27 Vesktop.app
---------------
Running phase: fixupPhase
```

</details>

> To me, a more correct way would be this as it wouldn't infer the filename based on the source directory:

Sure, it's a good practice